test/ussd_popi_faq.test.js

```diff
@@ -0,0 +1,597 @@
+var vumigo = require("vumigo_v02");
+var AppTester = vumigo.AppTester;
+var assert = require('assert');
+var fixtures_IdentityStore = require('./fixtures_identity_store');
+var fixtures_StageBasedMessaging = require('./fixtures_stage_based_messaging');
+var fixtures_MessageSender = require('./fixtures_message_sender');
+var fixtures_Hub = require('./fixtures_hub');
+var fixtures_Jembi = require('./fixtures_jembi');
+var fixtures_ServiceRating = require('./fixtures_service_rating');
+
+var utils = require('seed-jsbox-utils').utils;
+
+describe("app", function() {
+    describe("for ussd_popi_faq use", function() {
+        var app;
+        var tester;
+
+        beforeEach(function() {
+            app = new go.app.GoNDOH();
+
+            tester = new AppTester(app);
+
+            tester
+                .setup.config.app({
+                    name: 'ussd_popi_faq',
+                    env: 'test',
+                    metric_store: 'test_metric_store',
+                    testing_today: "2014-04-04 07:07:07",
+                    testing_message_id: '0170b7bb-978e-4b8a-35d2-662af5b6daee',
+                    logging: "off",
+                    no_timeout_redirects: [
+                        "state_start", "state_end_not_pregnant", "state_end_consent_refused",
+                        "state_end_success", "state_registered_full", "state_registered_not_full",
+                        "state_end_compliment", "state_end_complaint", "state_end_go_clinic"],
+                    channel: "*134*550*5#",
+                    services: {
+                        identity_store: {
+                            url: 'http://is/api/v1/',
+                            token: 'test IdentityStore'
+                        },
+                        stage_based_messaging: {
+                            url: 'http://sbm/api/v1/',
+                            token: 'test StageBasedMessaging'
+                        },
+                        hub: {
+                            url: 'http://hub/api/v1/',
+                            token: 'test Hub'
+                        },
+                        message_sender: {
+                            url: 'http://ms/api/v1/',
+                            token: 'test MessageSender'
```

Comment:
Thanks! I should have removed this when I removed it from the js file *facepalm*